web/src/flatpickr.ts

```diff
@@ -7,6 +7,12 @@ import assert from "minimalistic-assert";
 import {$t} from "./i18n";
 import {user_settings} from "./user_settings";
 
+export let flatpickr_instace: flatpickr.Instance;
```

Comment:
`instance` is mispelled.

Reply:
Added a commit to fix it since this commit was merged.